packaging/ios/build.sh

```diff
@@ -8,7 +8,7 @@ set -e
 TOP=$(pwd)
 SUBSURFACE_SOURCE=${TOP}/../../../subsurface
 IOS_QT=${TOP}/Qt
-QT_VERSION=$(cd ${IOS_QT}; ls -d 5.*)
+QT_VERSION=$(cd ${IOS_QT}; ls -d 5.*|perl -e 'print "5.".(sort {$b<=>$a} (map {/\d+\.(\d+.*)/; $1} <>))[0];')
```

Comment:
oh wow.
`QT_VERSION=$(cd ${IOS_QT}; ls -d [1-9]* | sort -n | tail -1)`
this will work even for Qt 6.0 and other variations.

Reply:
Yeah, except that sort -n thinks 5.10 < 5.8